platform/src/main/java/org/stellar/anchor/platform/service/PaymentOperationToEventListener.java

```diff
@@ -52,7 +50,7 @@ public void onReceived(ObservedPayment payment) {
     // Check if the payment contains the expected asset type
     if (!List.of("credit_alphanum4", "credit_alphanum12").contains(payment.getAssetType())) {
       // Asset type does not match
-      Log.warn("Not an issued asset");
+      Log.info("Not an issued asset");
```

Comment:
💡 nit: Can we add more event information here or at the beginning of the function to provide context of these messages?

Reply:
Sure, done in 8e0f8e30fa979abb4f1b2a4c1c1f45422a3c96a1.